azurelinuxagent/common/osutil/mariner.py

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+
+import os  # pylint: disable=W0611
```

Comment:
The pylint error here [unused-import<W0611>] basically means that this import is not being used in this file at all. Instead of ignoring the error, we should remove this import from this file. Same comment for all the imports which have the same error

Reply:
fixed